server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -1106,19 +816,23 @@ private IndexMetadata getIndexMetadata(String clusterName, String clusterUUID, U
      * @param clusterName name of the cluster
      * @return {@link IndexMetadata}
      */
-    public ClusterState getLatestClusterState(String clusterName, String clusterUUID) {
-        Optional<ClusterMetadataManifest> clusterMetadataManifest = getLatestClusterMetadataManifest(clusterName, clusterUUID);
+    public ClusterState getLatestClusterState(String clusterName, String clusterUUID) throws IOException {
+        start();
```

Comment:
start is being invoked in Node.java: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/node/Node.java#L1532. 
We dont need to call again.